src/ntimporters/monday/importer.py

```diff
@@ -0,0 +1,189 @@
+            if project.get("state") in ("archived", "deleted")
+            else None,
+            description=project.get("description"),
+            is_open=project.get("board_kind") == "public",
+            extra="",
+        )
+        nt_project = projects_api.post_project(strip_readonly(project_model)) or {}
+        if not (nt_project_id := str(nt_project.get("id"))):
+            return
+
+        _import_project_sections(nt_client, monday_client, nt_project_id, project, limits)
+
+    monday_projects = monday_client.projects()
+    monday_projects_open = [
+        elt.get("id") for elt in monday_projects if elt.get("board_kind") == "public"
+    ]
+    nt_projects = [
+        elt.get("id")
+        for elt in projects_api.get_projects()
+        if (elt.is_open and not hasattr(elt, "ended_at"))
+    ]
+    check_limits(
+        limits,
+        "projects_open",
+        len(monday_projects_open) + len(nt_projects),
+    )
+
+    for project in monday_projects:
+        _import_project(project)
+
+
+# pylint: disable=too-many-arguments
+def _import_project_sections(nt_client, monday_client, nt_project_id, project, limits: dict):
+    """Import monday lists as project sections"""
+    nt_api_sections = apis.ProjectSectionsApi(nt_client)
+
+    def _parse_timestamp(monday_timestamp: Optional[str]) -> Optional[models.TimestampNullable]:
+        """Parses monday timestamp into NT timestamp format"""
+        if not monday_timestamp:
+            return None
+        return models.TimestampNullable(int(isoparse(monday_timestamp).timestamp() * 1000))
+
+    check_limits(
+        limits,
+        "project_sections",
+        len(monday_sections := monday_client.sections(project.get("id"))),
+    )
+    sections_mapping = {}
+    for section in monday_sections:
+        if nt_section := nt_api_sections.post_project_section(
+            strip_readonly(
+                models.ProjectSection(
+                    models.Id16ReadOnly(id16()),
+                    models.Id16(nt_project_id),
+                    models.Name(section.get("title")),
+                    models.TimestampReadOnly(1),
+                    archived_at=models.TimestampReadOnly(1) if section.get("archived") else None,
+                    position=float(section.get("position") or 1.0),
+                )
+            )
+        ):
+            sections_mapping[section.get("id")] = str(nt_section.get("id"))
+    _import_tasks(nt_client, monday_client, sections_mapping, project.get("id"), nt_project_id)
+
+
+def _import_tasks(nt_client, monday_client, sections_mapping, m_project_id, nt_project_id):
+    """Import tasks"""
+    nt_api_tasks = apis.TasksApi(nt_client)
+    author_id = current_nt_member(nt_client)
+    for task in monday_client.tasks(m_project_id):
+        due_at, responsible_id = task.get("due_at"), None
+        if due_at:
+            due_at = models.TimestampNullable(due_at) if due_at else None
+            responsible_id = author_id
```

Comment:
Nozbe supports setting `responsible_id = "author"` - it would allow you to skip fetching author member for every project, although I'm not sure if it will pass validation

Reply:
i've added as TODO - to test it in the near ( ;-) ) future